kadasrouting/core/memorylayersaver.py

```diff
@@ -0,0 +1,365 @@
+"""
+credits to C. Crook for the original qgis MemoryLayerSaver plugin
+https://github.com/ccrook/QGIS-MemoryLayerSaver-Plugin
+"""
+
+# flake8: noqa
+# as this file was copied from https://github.com/ccrook/QGIS-MemoryLayerSaver-Plugin
+# we accept a bad formatting
```

Comment:
Even in this case, a little pass of a code formatter would be a nice thing, since format is quite inconsistent...

Reply:
I've done a bit of cleanup